tests/clients/test_experiment.py

```diff
@@ -363,3 +424,80 @@ def test_experiment_client_list_runs_page(mocker):
         schema_mock.return_value,
         params=[("start", 20), ("limit", 10)],
     )
+
+
+def test_log_run_data(mocker):
+    mocker.patch.object(ExperimentClient, "_patch_raw")
+    run_data_schema_mock = mocker.patch(
+        "faculty.clients.experiment.ExperimentRunDataSchema"
+    )
+    run_data_dump_mock = run_data_schema_mock.return_value.dump
+
+    client = ExperimentClient(mocker.Mock())
+    client.log_run_data(
+        PROJECT_ID,
+        EXPERIMENT_RUN_ID,
+        metrics=[METRIC],
+        params=[PARAM],
+        tags=[TAG],
+    )
+    expected_body = {"metrics": [METRIC], "params": [PARAM], "tags": [TAG]}
```

Comment:
I found the name here (`expected_body`) a little confusing as I anticipated that we'd be asserting that this was the actualy dispatched JSON, whereas we're actually asserting that this is the thing passed to the schema's dump method. Can we consider renaming to something like `expected_dump_arg` or just inline it into the `assert_called_once_with` below?

Reply:
👍